src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -86,12 +89,20 @@ export const PasswordAuthEntry = React.createClass({
             <div>
                 <p>To continue, please enter your password.</p>
                 <p>Password:</p>
-                <input
-                    ref="passwordField"
-                    className={passwordBoxClass}
-                    onChange={this._onPasswordFieldChange}
-                    type="password"
-                />
+                <form onSubmit={this._onSubmit}>
+                    <input
+                        ref="passwordField"
+                        className={passwordBoxClass}
+                        onChange={this._onPasswordFieldChange}
+                        type="password"
+                    />
+                    <div className="mx_button_row">
+                        <input type="submit"
+                            className="mx_Dialog_primary"
```

Comment:
Using the mx_Dialog class here: the button classes need refactoring into a single class, so rather than c+p the CSS yet again, I'm reusing this class.